recipes/conda/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "4.1.5" %}
+
+package:
+  name: conda
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/conda/conda
+  git_tag: {{ version }}
+  git_depth: 1
```

Comment:
Seems the cloning issue can be resolved by using 2 commits for our shallow clone instead of 1. Go figure. Honestly, I'm not interested in going down this rabbit hole any further as long as it works. If anyone else wants to understand it, be my guest.